src/lib/libast/tests/path/pathgetlink.c

```diff
@@ -0,0 +1,42 @@
+#include "config_ast.h"  // IWYU pragma: keep
+
+#include <errno.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+
+#include "ast.h"
+#include "terror.h"
+
+tmain() {
+    UNUSED(argc);
+    UNUSED(argv);
+    const char *linkname = "foo";
+
+    char buff[10];
+    char filename[] = "foobarbaz";
```

Comment:
Why isn't this `const char *filename = "foobarbaz";`?

Reply:
I was playing around with some other functions that required this variable to be writable, but will fix it.